run.py

```diff
@@ -510,7 +517,16 @@ def run(command, env={}, log_filename={}):
                     run_fmri = True
                     print("     ... fMRI config created : {}".format(
                         project.fmri_config_file))
-
+                if args.eeg_pipeline_config is not None:
+                    if check_configuration_format(args.func_pipeline_config) == '.ini':
+                        eeg_pipeline_config = convert_config_ini_2_json(args.eeg_pipeline_config)
+                    else:
+                        eeg_pipeline_config = args.eeg_pipeline_config
+                    project.eeg_config_file = create_subject_configuration_from_ref(project, eeg_pipeline_config,
+                                                                                     'eeg')
+                    run_eeg = True
+                    print("     ... EEG config created : {}".format(
+                        project.eeg_config_file))
```

Comment:
```suggestion
                    print(f'     ... EEG config created : {project.eeg_config_file}')
```

Reply:
f-string starting from python 3.4 or 3.5 are great to make print of variables more compact and readable than format